composer.json

```diff
@@ -17,13 +17,14 @@
     "require": {
         "php":                  ">=5.3.6",
         "behat/mink":           "^1.7.1@dev",
-        "symfony/browser-kit":  "~2.3|~3.0",
-        "symfony/dom-crawler":  "~2.3|~3.0"
+        "symfony/browser-kit":  "~2.3|~3.0|~4.0",
+        "symfony/dom-crawler":  "~2.3|~3.0|~4.0"
     },
 
     "require-dev": {
         "mink/driver-testsuite": "dev-master",
-        "symfony/http-kernel": "~2.3|~3.0"
+        "symfony/http-kernel": "~2.3|~3.0|~4.0",
+        "phpunit/phpunit": "^4.8.35|^5.7"
```

Comment:
this requirement does not make sense if you use `vendor/bin/simple-phpunit`

Reply:
That's a good point. Removed.